src/components/tables/BuildTable.vue

```
@@ -0,0 +1,207 @@
+<script setup lang="ts">
+import type { Ref } from 'vue'
+import type { TableColumn } from '../comp_def'
+import type { Database } from '~/types/supabase.types'
+import dayjs from 'dayjs'
```

ESLint

[error] 'dayjs' is defined but never used.

(unused-imports/no-unused-imports)